distribution/tests/endpoints_ps.ps1

```diff
@@ -0,0 +1,28 @@
+Get-WmiObject Win32_Process -filter "CommandLine LIKE '%alfresco-identity-service-$IDENTITY_VERSION%'"
+
+./standalone.ps1 --background
+
+start-sleep -s 15
+
+function checkStatus {
+    if ($args[0] -notmatch $args[1]) {
+        Get-WmiObject Win32_Process -filter "CommandLine LIKE '%alfresco-identity-service-$IDENTITY_VERSION%'" | foreach { kill $_.ProcessId }
+        $test=$args[0]
+        $check=$args[1]
+        throw "Accessing $test does not output $check "
+    }
+}
+
+checkStatus (curl -v http://localhost:8080/auth/).StatusCode "200"
```

Comment:
Does curl need to be pre-installed on a Windows box in order for this to work?

Reply:
Nope, the curl that you actually see here is an alias for Invoke-WebRequest, a command that comes with powershell. https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/Invoke-WebRequest?view=powershell-4.0

And returns an object out of which i only get the StatusCode property.